lib/browser/api/app.js

```diff
@@ -58,6 +58,13 @@ Object.assign(app, {
   }
 })
 
+const nativeFn = app.getAppMetrics
+app.getAppMetrics = () => {
+  deprecate.removeProperty(nativeFn, 'privateBytes')
+  deprecate.removeProperty(nativeFn, 'sharedBytes')
```

Comment:
It looks slightly strange to me that the deprecation messages are printed for every method call, but I guess it's the best we can do here.